config/ec2/config.go

```diff
@@ -285,6 +285,11 @@ func Configure(p *config.Provider) {
 		// Mutually exclusive with:
 		// vpc_endpoint_service_allowed_principal
 		config.MoveToStatus(r.TerraformResource, "allowed_principals")
+		r.References["network_load_balancer_arns"] = config.Reference{
```

Comment:
Looks like `Type:` is missing? Please consult with https://github.com/upbound/upjet/blob/main/docs/add-new-resource-long.md#cross-resource-referencing

Reply:
@ytsarev If I add LB reference in Type:, It will cause cyclic import between EC2 & ELBv2 modules

Reply:
https://github.com/upbound/upjet/issues/96